core/src/com/mygdx/crazybomber/MainRooms.java

```diff
@@ -0,0 +1,136 @@
+    private TextureAtlas atlas;
+    private Sprite background;
+    private SpriteBatch batch;
+    private MapRepository mapRepository;
+    private RoomRepository roomRepository;
+    private StateRepository stateRepository;
+
+    public MainRooms(MapRepository mapRepository, RoomRepository roomRepository, StateRepository stateRepository) {
+        this.mapRepository = mapRepository;
+        this.roomRepository = roomRepository;
+        this.stateRepository = stateRepository;
+    }
+    @Override
+    public void show() {
+        batch = new SpriteBatch();
+        background = new Sprite(new Texture("background1.png"));
+        background.setSize(Gdx.graphics.getWidth(),Gdx.graphics.getHeight());
+
+        stage = new Stage();
+        Gdx.input.setInputProcessor(stage);
+        atlas = new TextureAtlas("ui/button.pack");
+        skin = new Skin(Gdx.files.internal("menuSkin.json"),atlas);
+        table = new Table(skin);
+        table.setBounds(0,0,Gdx.graphics.getWidth(),Gdx.graphics.getHeight());
+
+        buttonRoom1 = new TextButton("ROOM1",skin);
+        buttonRoom1.setWidth(1000);
+        buttonRoom1.addListener(new ClickListener(){
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                ((Game)Gdx.app.getApplicationListener()).setScreen(roomRepository.getRoom(0));
+            }
+        });
+        buttonRoom2 = new TextButton("ROOM2",skin);
+        buttonRoom2.addListener(new ClickListener(){
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                ((Game)Gdx.app.getApplicationListener()).setScreen(roomRepository.getRoom(1));
+            }
+        });
+        buttonRoom3 = new TextButton("ROOM3",skin);
+        buttonRoom3.addListener(new ClickListener(){
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                ((Game)Gdx.app.getApplicationListener()).setScreen(roomRepository.getRoom(2));
+            }
+        });
+        buttonBack = new TextButton("BACK",skin);
+        buttonBack.addListener(new ClickListener(){
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                stateRepository.pop();
+                ((Game) Gdx.app.getApplicationListener()).setScreen(stateRepository.peek());
+            }
+        });
+        heading =  new Label("Rooms", skin);
+        heading.setFontScale(2);
+        table.add(heading);
+        table.row();
+        table.row();
+        table.padBottom(50);
+        table.add(buttonRoom1);
+        table.getCell(buttonRoom1).size(1000,150);
+        table.row();
+        table.add(buttonRoom2);
+        table.getCell(buttonRoom2).size(1000,150);
+        table.row();
+        table.add(buttonRoom3);
+        table.getCell(buttonRoom3).size(1000,150);
+        table.row();
+        table.add(buttonBack).bottom().left();
+        table.debug();
```

Comment:
what does this do?

Reply:
Make sure to delete it from all classes that use table if you decide to delete it.

Reply:
it's to place where the back button goes on the screen